tbx/blog/tests/test_models.py

```diff
@@ -0,0 +1,127 @@
+from faker import Faker
+from tbx.blog.factories import BlogIndexPageFactory, BlogPageFactory
+from tbx.blog.models import BlogPage
+from tbx.core.factories import HomePageFactory
+from tbx.people.factories import AuthorFactory
+from tbx.taxonomy.factories import ServiceFactory
+from tbx.taxonomy.models import Service
+from wagtail.models import Site
+from wagtail.test.utils import WagtailPageTestCase
+from wagtail.test.utils.form_data import (
+    inline_formset,
+    nested_form_data,
+    rich_text,
+    streamfield,
+)
+
+fake = Faker(["en_GB"])
+
+
+class TestBlogIndexPageFactory(WagtailPageTestCase):
+    def test_create(self):
+        BlogIndexPageFactory()
+
+
+class TestBlogPageFactory(WagtailPageTestCase):
+    def test_create(self):
+        blog_post = BlogPageFactory()
+        self.assertEqual(blog_post.related_services.count(), 1)
+
+        services = ServiceFactory.create_batch(size=3)
+        another_blog_post = BlogPageFactory(related_services=list(services))
+        self.assertEqual(another_blog_post.related_services.count(), 3)
+
+
+class TestBlogPage(WagtailPageTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        site = Site.objects.get(is_default_site=True)
+        root = site.root_page.specific
+        home_page = HomePageFactory(parent=root)
+
+        site.root_page = home_page
+        site.save()
+
+        cls.blog_index = BlogIndexPageFactory(parent=home_page, title="Blog")
+
+    def test_can_create_blog_page_with_supplied_post_data(self):
+        service = ServiceFactory()
+        author = AuthorFactory()
+        self.login()
+        self.assertCanCreate(
+            self.blog_index,
+            BlogPage,
+            nested_form_data(
```

Comment:
Is this test similar to the standard page one that @ababic thought might be overkill, as it is the same as testing wagtail core functionality? If we don't need a test we should avoid it as the more tests we have, the slower the pipeline.

Reply:
I have removed superflous tests in 822f66c